packages/libiconv.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Libiconv < Package
+  description 'GNU charset conversion library for libc which does not implement it.'
+  homepage 'https://www.gnu.org/software/libiconv/'
+  version '1.15'
+  source_url 'https://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.15.tar.gz'
```

Comment:
Does `https://ftpmirror.gnu.org/libiconv/libiconv-1.15.tar.gz` work?  This would find the closest mirror site.

Reply:
wget https://ftpmirror.gnu.org/libiconv/libiconv-1.15.tar.gz
--2018-03-12 01:46:22--  https://ftpmirror.gnu.org/libiconv/libiconv-1.15.tar.gz
Resolving ftpmirror.gnu.org... 208.118.235.200
Connecting to ftpmirror.gnu.org|208.118.235.200|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://mirror.ibcp.fr/pub/gnu/libiconv/libiconv-1.15.tar.gz [following]
--2018-03-12 01:46:24--  https://mirror.ibcp.fr/pub/gnu/libiconv/libiconv-1.15.tar.gz
Resolving mirror.ibcp.fr... 2001:660:500b:245::138, 194.254.245.138
Connecting to mirror.ibcp.fr|2001:660:500b:245::138|:443... connected.

Reply:
Can we use that url instead?  It will likely be faster in other parts of the world including China. :)

Reply:
Yes, 'no problem'.